swag/yfu.py

```diff
@@ -79,7 +80,7 @@ def on_loss(self):
 
 
 # TODO move this class
-class YfuEmbed(discord.Embed):
+class YfuEmbed(disnake.Embed):
```

Comment:
I did see another YfuEmbed somewhere, no?

Reply:
Wtf

Reply:
Ah yes, I just forgot to delete it.